components/tikv_util/src/metrics/threads_dummy.rs

```diff
@@ -7,9 +7,10 @@ other than Linux. PRs are welcome!
 
 */
 
-use std::collections::HashMap;
 use std::io;
 
+use crate::collections::HashMap;
```

Comment:
I find some other places also use std HashMap, can you send a PR later to fix them all?

Reply:
OK